test/run/ch-fromhost.bats

```diff
@@ -26,194 +26,194 @@ fromhost_ls () {
 @test 'ch-fromhost (Debian)' {
     scope standard
     prerequisites_ok debian9
-    IMG=$IMGDIR/debian9
+    img=$ch_imgdir/debian9
```

Comment:
Should use the one from `common.bash`?

Reply:
ch_img uses `ch_tag=$(basename "$BATS_TEST_DIRNAME")`, which I think for ch-fromhost.bats will be `run` (correct me if I am wrong). I think the local img variable is appropriate here.